dot/types/validator.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package types
```

Comment:
should we have tests for this file?

Reply:
Forgot to remove the methods in this. Thank you.

It doesn't need tests now.